db/seeds/020_organization_seeds.rb

```diff
@@ -1,9 +1,15 @@
 # frozen_string_literal: true
 
-Organization.new(name: 'seed-user-organization',
-                 display_name: 'Seed User Organization',
-                 description: 'All users that are created in the seeds').save
+Organization.
+  new(name: 'seed-user-organization',
+      display_name: 'Seed User Organization',
+      description: 'All users that are created in the seeds').save.tap do |org|
```

Comment:
Please use `.create(...)` instead of `.new(...).save`. `.create(...)` is the same as `.new(...).save` except for the returned value: `save` returns a boolean while `create` returns the created object. So with `save`, the `tap` should not work as intended.